Src/BootCamp.Chapter/Demo/DemoDelegatesAndEvents.cs

```diff
@@ -0,0 +1,55 @@
+﻿using BootCamp.Chapter.Events;
+using System;
+using System.Transactions;
+
+namespace BootCamp.Chapter.Demo
+{
+    public static class DemoDelegatesAndEvents
+    {
+        private static event EventHandler<ApplicationStateEventArgs> ApplicationStateHandler;
```

Comment:
It's not a handler, is an event. OnApplicationStarted for example. You will call this event, all those that subscribed to it will be hit.